src/plugins/visualizations/public/types.ts

```diff
@@ -36,9 +36,40 @@ export interface ISavedVis {
   uiStateJSON?: string;
   savedSearchRefName?: string;
   savedSearchId?: string;
+  sharingSavedObjectProps?: {
+    outcome?: 'aliasMatch' | 'exactMatch' | 'conflict';
+    aliasTargetId?: string;
+    errorJSON?: string;
+  };
 }
 
-export interface VisSavedObject extends SavedObject, ISavedVis {}
+export interface VisSavedObject extends ISavedVis {
+  lastSavedTitle: string;
+  copyOnSave?: boolean;
```

Comment:
honestly for me it looks more like `SaveVisOptions`.  The same question around `isSaving`.

Reply:
agree, moved copyOnSave in SaveVisOptions